.github/workflows/ci.yml

```
@@ -130,28 +121,25 @@ jobs:
-        if: matrix.node-version == 22
         run: pnpm --filter @bluelight-hub/frontend exec playwright install --with-deps chromium
 
-      - name: Run E2E tests
-        if: matrix.node-version == 22
-        env:
-          CI: true
-        run: pnpm --filter @bluelight-hub/frontend test:e2e:auth
+        #      - name: Run E2E tests
+        #        env:
+        #          CI: true
+        #run: pnpm --filter @bluelight-hub/frontend test:e2e:auth
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)